installer/macos/create_pkg.sh

```diff
@@ -38,20 +48,31 @@ install_massastation_build_dependencies() {
 build_massastation() {
     install_massastation_build_dependencies
 
-    go generate ../... || fatal "go generate failed for $MASSASTATION_INSTALLER_NAME"
+    go generate ../... || fatal "go generate failed for $MASSASTATION_APPLICATION_NAME"
     export GOARCH=$ARCH
     export CGO_ENABLED=1
     # -icon is based on the path of the -src flag.
-    fyne package -icon ../../int/systray/embedded/logo.png -name MassaStation -appID com.massalabs.massastation -src ../cmd/massastation || fatal "fyne package failed for $MASSASTATION_INSTALLER_NAME"
-    chmod +x $MASSASTATION_INSTALLER_NAME || fatal "failed to chmod $MASSASTATION_INSTALLER_NAME"
+    fyne package -icon ../../int/systray/embedded/logo.png -name MassaStation -appID com.massalabs.massastation -src ../cmd/massastation || fatal "fyne package failed for $MASSASTATION_APPLICATION_NAME"
+    chmod +x $MASSASTATION_APPLICATION_NAME || fatal "failed to chmod $MASSASTATION_APPLICATION_NAME"
 }
 
 # Build the package using pkgbuild.
 package() {
-    pkgbuild --component $MASSASTATION_INSTALLER_NAME --identifier com.massalabs.massastation --version $PKGVERSION \
+    # sign the application if we have a developer id
+    if [[ -n "$APPLE_DEVELOPER_ID_APPLICATION" ]]; then
+        codesign --force --options runtime --sign "$APPLE_DEVELOPER_ID_APPLICATION" $MASSASTATION_APPLICATION_NAME
```

Comment:
Why `--force` ? 🤔 

(non blocking)

Reply:
to replace any signature already there, it fails if there is already a signature and we don't use the --force option. However, there should not be a signature already but let's be sure.